demo/bicep/cluster.bicep

```diff
@@ -1,80 +1,464 @@
+// TODO
+// * KMS identity + role assignments need to be created
+// * pass the KMS identity to the cluster
+
```

Comment:
can we make each of these separate bicep templates so we can easily test good and bad scenarios?

Reply:
we can address that in future changes. makes sense for sure